rinex/src/tests/compression.rs

```diff
@@ -1,7 +1,8 @@
 #[cfg(test)]
 mod test {
-    use rinex::prelude::*;
-    use rinex::test_toolkit;
+    use crate::prelude::*;
+    use crate::tests::toolkit::*;
+    use std::path::PathBuf;
     //#[test]
     fn crinex1() {
```

Comment:
Is this intentionally disabled?

Reply:
Yes, there are still some issues in the RNX->CRNX operation.  
This test partially passes, and partially fails, and at the time I decided to temporarily disable it.
Will get back to file operations when I've accomplished what I want with file processing

Reply:
You can use `#[ignore]`, it's a little more explicit.

Reply:
that's a cool asset, thanks

Reply:
Resolving this conversation to facilitate merging :)